commitlint.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  parserPreset: './parser-preset.js',
+  rules: {
+    'type-enum': [2, 'always', [`FRDQ`]],
+    'commit-message-rule': [2, 'always'],
+    'subject-empty': [2, 'never'],
+    'subject-case': [2, 'always', 'sentence-case'],
+  },
+  plugins: [
+    {
+      rules: {
+        'commit-message-rule': ({ type, id }) => {
+          const PREFIX = `FRDQ`;
+          const ID = parseInt(id, 10);
+
+          return [
+            !isNaN(ID) && type.includes(`${PREFIX}`),
```

Comment:
Could you return more customized messages?
ID is not a number
PREFIX is not correct